sway/commands/output/toggle.c

```diff
@@ -0,0 +1,37 @@
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/output.h"
+
+struct cmd_results *output_cmd_toggle(int argc, char **argv) {
+    if (!config->handler_context.output_config) {
+        return cmd_results_new(CMD_FAILURE, "Missing output config");
+    }
+
+    struct output_config *output = config->handler_context.output_config;
+
+    if (argc > 1 && strcmp(output->name, "*") == 0) {
+        return cmd_results_new(CMD_INVALID,
+                "Cannot apply toggle to all outputs.");
+    }
+
+    struct sway_output *s_output = all_output_by_name_or_id(output->name);
```

Comment:
style nit: sway_output or just output (renaming the other variable to output_config or oc) would be better.